<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XfstkDldrPluginOptionsInterface</class>
 <widget class="QWidget" name="XfstkDldrPluginOptionsInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QTabWidget" name="OptionsInterface">
       <property name="tabPosition">
        <enum>QTabWidget::South</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>CLV B0/C0</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QTabWidget" name="tabWidget">
           <property name="currentIndex">
            <number>3</number>
           </property>
           <widget class="QWidget" name="tab_5">
            <attribute name="title">
             <string>General</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_12">
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_11">
               <item row="0" column="0">
                <widget class="QGroupBox" name="groupBox_2">
                 <property name="title">
                  <string>Provision Count</string>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_13">
                  <item row="0" column="0">
                   <layout class="QGridLayout" name="gridLayout_8">
                    <item row="0" column="0">
                     <widget class="QCheckBox" name="ProvisionCountEnabled">
                      <property name="text">
                       <string>Display Provisions Attempted Count</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="1">
                     <widget class="QPushButton" name="ResetProvisionCount">
                      <property name="text">
                       <string>Reset Count</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </item>
             <item row="2" column="0">
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_4">
            <attribute name="title">
             <string>Logging</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_10">
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_9">
               <item row="0" column="0">
                <widget class="QCheckBox" name="EnableLogging">
                 <property name="text">
                  <string>Log Debug Output to a File</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>Log Directory</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLineEdit" name="LoggingPath"/>
               </item>
               <item row="2" column="1">
                <widget class="QToolButton" name="BrowseLoggingPath">
                 <property name="text">
                  <string>...</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="0">
              <widget class="QPushButton" name="RestoreDefaultLogPath">
               <property name="text">
                <string>Restore Default Directory</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_2">
            <attribute name="title">
             <string>Stored Paths</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_3">
               <item row="0" column="1">
                <widget class="QLineEdit" name="FWDnXPath">
                 <property name="text">
                  <string>N/A</string>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>FW DnX:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QPushButton" name="ClearFWDnXPath">
                 <property name="text">
                  <string>Clear</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="IFWIPath">
                 <property name="text">
                  <string>N/A</string>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>IFWI:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QPushButton" name="ClearIFWIPath">
                 <property name="text">
                  <string>Clear</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLineEdit" name="OSDnXPath">
                 <property name="text">
                  <string>N/A</string>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="2" column="2">
                <widget class="QPushButton" name="ClearOSDnXPath">
                 <property name="text">
                  <string>Clear</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="2">
                <widget class="QPushButton" name="ClearOSImagePath">
                 <property name="text">
                  <string>Clear</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QLineEdit" name="OSImagePath">
                 <property name="text">
                  <string>N/A</string>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>OS DnX:</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>OSImage:</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="0">
              <widget class="QPushButton" name="ClearAllStoredPaths">
               <property name="text">
                <string>Clear All Paths</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_3">
            <attribute name="title">
             <string>Download Options</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_7">
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_6">
               <item row="0" column="0">
                <widget class="QCheckBox" name="GPFlagOverrideEnable">
                 <property name="text">
                  <string>Enable GP Flag Override</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>GP Flag Override Value:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="GPFlagOverrideValue"/>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>Example: 0x80000001</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="3" column="1">
                <widget class="QLineEdit" name="UsbTimeoutOption">
                 <property name="text">
                  <string>60</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>Device Detection Timeout</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
